lib/matplotlib/rcsetup.py

```diff
@@ -175,6 +175,21 @@ def validate_string_or_None(s):
         raise ValueError('Could not convert "%s" to string' % s)
 
 
+def validate_stringlist_or_string(s):
```

Comment:
make this private (start with underscore)

Reply:
Is that necessary? All other validate_something() functions don't begin with an underscore.

Reply:
Things that are currently public API we can not claw back, but we would like to avoid adding to it if possible.

Reply:
see b7f7d06